### ISSUE TITLE
Replicate V1.5 feedback for matching results

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#1221]
In V1.5 when the program was finished matching records, it produced a modal that said how many records matched, etc.
In V 2.0 right now, we are only reporting the duplicates (and I am not sure that is accurate).
Need to replicate the functionality of V 1.5 -- I will provide screen shots of V 1.5 modal when we get to this.
Number of duplicates not imported
Number of matches
Number of records imported
etc

**Comment by axelstudios** [comment]
The v2.0 matching results dialog has all of the data but shows different things in certain conditions (details about duplicates were removed entirely).  We can add more stats though:

![image](https://cloud.githubusercontent.com/assets/411466/22353770/5215914c-e3df-11e6-9aa1-c467770cbb01.png)



![image](https://cloud.githubusercontent.com/assets/411466/22353817/958f55ca-e3df-11e6-82cb-93848a3c049d.png)

**Comment by RDmitchell** [comment]
Why were details about duplicates removed? That is actually important
information to give to the user when they are importing PM files that have
previously imported data that we don't want to import again.

**Comment by axelstudios** [comment]
Functionally, there's no difference between a "duplicate" and a "match", so it only reports on matching results.

**Comment by RDmitchell** [comment]
While that may be true, I don't think we want to be accumulating all the duplicate Portfolio Manager records (that accumulate in the CSV download from PM as new data is added) in this history. 

Some users want to download this PM file once a day or more often (and it could be even more often than that if we automate the download), and so the history for one record would become completely overloaded with the exact same entry every time that happens. 

So in V 1.5, we compared every field when matching (in this case merging) and if there was already the EXACT same record (ie, all the data plus same PM Generation Date and PM Release Date) we did not include it in the merge. Before we implemented that, the columns in the Detail view were getting out of hand with the exact same data in them. 

I think we should keep this same philosophy in 2.0 -- there is no need to fill up the audit / history table (and the columns in the detail view) with identical records. 

I suppose an argument could be made to keep them in the audit/history table, but not view them in the detail view, but I guess it should be considered whether the exact same record that is imported over and over again actually needs to be stored. 

Also, I think it will be useful to the user to know how many records actually represent changed data when merged, rather than just the exact same data that was already imported previously. The users are trying to see what building owners amended their PM data, so that is the type of feedback we should be giving them in this screen. When we reported the duplicates in V 1.5, we were reporting to the user the number of records that were in the file being merged that were already there and were not added again with no new data in them.
